docs/content/preview/secure/enable-authentication/ysql.md

```diff
@@ -7,7 +7,7 @@ headcontent: Enable Users in YSQL.
 image: /images/section_icons/secure/authentication.png
```

Comment:
title: Enable Users in YSQL
headerTitle: Enable Users in YSQL
linkTitle: Enable Users in YSQL
description: Enable Users in YSQL.
headcontent: Enable Users in YSQL.

change to 
title: Enable users in YSQL
headerTitle: Enable users in YSQL
linkTitle: Enable users in YSQL
description: Enable users in YSQL.
headcontent: Enable users in YSQL.